engine/events/battle_tower/battle_tower.asm

```diff
@@ -197,7 +199,7 @@ _BattleTowerBattle:
 	call .do_dw
 	call DelayFrame
 	ld a, [wBattleTowerBattleEnded]
-	cp $1
+	cp 1
```

Comment:
`cp TRUE`

Reply:
wInBattleTowerBattle is a weird one, it's used as bitflags (through the `bit` instruction) in a lot of places, so it's most definitely not a boolean.

Reply:
Even more so if it's a bitflag, then please leave related values as hex until they have `*_F` constants identified.